scripts/web.xml

```diff
@@ -2,7 +2,7 @@
 <web-app>
     <servlet>
         <servlet-name>JsonRpcServlet</servlet-name>
-        <servlet-class>kbaserelationengine.KBaseRelationEngineServer</servlet-class>
+        <servlet-class>kbasesearchengine.KBaseSearchEngineServer</servlet-class>
```

Comment:
Shouldn't there some sort of start immediately directive here? I seem to remember something like that.

Reply:
Yes it should. Putting it in.